grunt/watch.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
     templates: {
         options: { cwd: "<%= ngtemplates.build.cwd %>" },
         files: "<%= ngtemplates.build.src %>",
```

Comment:
These lines here need changed, otherwise saving a template file doesn't do anything. They should reference `svg_inline`'s `cwd`, and `src`